moml/data/dataset.py

```
@@ -0,0 +1,51 @@
+from typing import Optional, Union, Callable
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)